cmake/Modules/FindMKLDNN.cmake

```diff
@@ -18,6 +18,47 @@ IF(NOT MKLDNN_FOUND)
 
   SET(IDEEP_ROOT "${PROJECT_SOURCE_DIR}/third_party/ideep")
   SET(MKLDNN_ROOT "${PROJECT_SOURCE_DIR}/third_party/ideep/mkl-dnn")
+
+  if(USE_XPU) # Build oneDNN GPU library
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+      set(DNNL_HOST_COMPILER "g++")
+      # g++ is soft linked to /usr/bin/cxx, oneDNN would not treat it as an absolute path
+    else()
+      set(DNNL_HOST_COMPILER DEFAULT)
+    endif()
+
+    set(MAKE_COMMAND "cmake" "--build" ".")
```

Comment:
Hi, @malfet  great thanks for the comments. 
The variable name is confusing here, the `MAKE_COMMAND` here is an argument required for `ExternalProject_Add` and it is expected to be used only for oneDNN compilation. Also, it would not override variables in other part. I will modify it to `XPU_DNNL_MAKE_COMMAND` for avoiding confusion.

Reply:
@malfet Sincere thanks for you comments again, I've change the MAKE_COMMAND to `DNNL_MAKE_COMMAND` for avoid confusion in current commit